tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -103,6 +105,10 @@ def _run(self):
                 )
                 raise ex
 
+        # instantiating blivet class, this will be used for
+        # getting brick_device_details
+        b = blivet.Blivet()
+
         while not self._complete.is_set():
```

Comment:
Please rebase on top of https://github.com/Tendrl/gluster-integration/pull/352/files